README.md

```diff
@@ -28,7 +28,8 @@ Use the following markup to enable a comment stream:
 <div class="o-comments"
 	id="o-comments-stream"
 	data-o-component="o-comments"
-	data-o-comments-article-id="{article-id}">
+	data-o-comments-article-id="{article-id}"
+    data-o-comments-story-url="{optional-story-url}">
```

Comment:
Everything looks good except the indentation is off here and below with the storyUrl.

Reply:
Good catch. Thanks!